app/utils/configuration/auth.py

```diff
@@ -0,0 +1,29 @@
+from configparser import ConfigParser
+
+# TODO - seemed like a good idea but is a little messier that I'd envisoned
+# try it in anger for a bit but don't be afraid to refactor.
+class AuthConfig:
+    """
+    Convenience for neater auth releveant variables taken or
+    extrapolated from the wider app config
+    """
+
+    def __init__(self, cfg: ConfigParser):
+        oauth_domain = cfg.get('ENV', 'oauth_domain')
+
+        # Auth related standard config
+        self.expiry_minutes = int(cfg.get('AUTH', 'expiry_minutes'))
+        self.algorithm = cfg.get('AUTH', 'jwt_encoding_algorithm')
+        self.roles_namespace = cfg.get('AUTH', 'roles_namespace')
+
+        # Things that started as env vars
+        self.encryption_key = cfg.get('ENV', 'encryption_key')
+        self.client_id = cfg.get('ENV', 'oauth_client_id')
+        self.client_secret = cfg.get('ENV', 'oauth_client_secret')
+        self.redirect_uri = cfg.get('ENV', 'oauth_redirect_uri')
+
+        # Extrapolated
+        self.access_token_url = f'https://{oauth_domain}/oauth/token'
```

Comment:
RE: client_secret, client_id, encryption_key - rather than having this in 'ENV' when we go to prod, perhaps have this as a keystorage solution in the cloud when we get further along with it

Reply:
re: file naming convention - possibly think about changing, doesnt conflict with app/auth.py